frontend/src/components/static_pages/about.css

```diff
@@ -0,0 +1,43 @@
+.images {
+  width: 400px;
+  margin: 5%;
```

Comment:
hmm i think this would be better with a fixed `px` maybe? 

also may want to move this class up a level to the div and not just on the img tag. and it might layout a bit better

Reply:
I tried that, but the image is taking the entire width of the container.
![Screenshot](https://user-images.githubusercontent.com/28938627/62504239-38228880-b7bd-11e9-85da-7a2bcc6714a1.png)

Reply:
So if you move up the class name you will have to re define the image width so something like

```
.images { magin: 5px }
.images img { width: 400px }
```

Reply:
you also may not have to move the class name up for this but just do another class name for the parent and set the margin there. 

something like 

```
.image-container { margin: 5px } 
```